### ISSUE TITLE
Does not work with `silverstripe/framework: ^4`

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-mssql#46]
the move from site root `modulename/` modules to `vendor/vendorname/modulename/` convention has seen this module fall behind in functional status.

**Comment by kinglozzer** [comment]
Looks like https://github.com/silverstripe/silverstripe-mssql/pull/47 resolved this?

**Comment by robbieaverill** [comment]
It'd be great if you could provide some more specifics about what's not working. The issue mentions a problem with the vendor module status and #47 is related to that.

What's remaining in terms of problems after these other PRs?

**Comment by NightJar** [comment]
The issue is intentionally vague as I couldn't even install the module before, therefore had no way of telling what was wrong with it (beyond a bit of a lack of namespaces).
This mostly remains true, as it cannot actually be used to build a site in order to give it a thorough testing.

Current error status :
```
[Alert] Call to undefined method SilverStripe\MSSQL\MSSQLSchemaManager::parseIndexSpec()
GET /dev/build
```

**Comment by kinglozzer** [comment]
Yep, sorry - #51 gets the installer running to resolve the `parseIndexSpec()` issue. There’s then another fix required to transactions to get everything else running smoothly: https://github.com/kinglozzer/silverstripe-mssql/commit/586dac5358f792adb4a94e4d4c137027bff443e1.

I actually don’t have a Windows environment to test on at the moment, I was working through these issues with the help of someone via email! They’ve confirmed that everything is working for them after the two fixes above are applied

**Comment by NightJar** [comment]
I am one of the few devs who does run windows it seems, which is why I began this quest after a community member pointed out it was broken. I'll test your work in #51 shortly :)

**Comment by NightJar** [comment]
Closed by combination of #47 and #51 
Test evidence at:https://github.com/silverstripe/silverstripe-mssql/pull/51#issuecomment-392729741
